src/behaviors.js

```diff
@@ -47,7 +47,7 @@ Marionette.Behaviors = (function(Marionette, _) {
         behaviorEvents = Marionette.normalizeUIKeys(behaviorEvents, getBehaviorsUI(b));
 
         var j = 0;
-        _.each(behaviorEvents, function(behaviour, key) {
+        _.each(behaviorEvents, function(behaviorEvent, key) {
```

Comment:
I think the actual event is the 'key' though. In this case 'behaviorEvent'
is what actually responds to the event.

On Wednesday, 5 August 2015, W. van Kuipers <notifications@github.com>
wrote:

> In src/behaviors.js
> <https://github.com/marionettejs/backbone.marionette/pull/2687#discussion_r36272814>
> :
>
> > @@ -47,7 +47,7 @@ Marionette.Behaviors = (function(Marionette, _) {
> >          behaviorEvents = Marionette.normalizeUIKeys(behaviorEvents, getBehaviorsUI(b));
> >
> >          var j = 0;
> > -        _.each(behaviorEvents, function(behaviour, key) {
> > +        _.each(behaviorEvents, function(behaviorEvent, key) {
>
> It's a item from the behaviorEvents collection, so behaviorEvent seems to
> be the correct phrase. Handler could also be working, but I find it a bit
> more abstract.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/marionettejs/backbone.marionette/pull/2687/files#r36272814>
> .
>